### PR TITLE
PRTL-624 ✨ angular githut in react

### PR DESCRIPTION
**Pull request by cheapsteak**
image

git hut

### CODE REVIEW

**Comment by Jephuff** [comment]
Click events aren't working for me, is it possible to pass handlers in?

**Comment by cheapsteak** [comment]
Ah good catch, I didn't realize it was clickable
It seems to directly update location, but I neutred angular's location.,... I think I can fix that  :)

File: modules/node_modules/@ncigdc/components/GitHut.js [hunk 1]
```diff
@@ -64,6 +69,22 @@ class GitHut extends React.Component {
     }
   }
 
+  handleAngularRouterRequestGo = (state, params, options) => {
```
Comment (cheapsteak) [inline modules/node_modules/@ncigdc/components/GitHut.js]: I am open to suggestions for what to name this
It handles when angular's router requests a ui-router state change

**Comment by cheapsteak** [comment]
@Jephuff the hut(s) are now clickable!
We're now hijacking ui-router's $state.go and when the angular app wants to go somewhere, we're handling the request and dispatching transformed params to react-router :D

**Comment by Jephuff** [comment]
hmm, I'm getting an error because params.filter is not defined in `handleAngularRouterRequestGo`?

**Comment by cheapsteak** [comment]
Having trouble reproducing, could you please dump the stacktrace?

**Comment by cheapsteak** [comment]
Should be fixed! Now accounting for clicking on project ids

**Comment by Jephuff** [comment]
nice 👍

**Pull request merged**